EasyPost.Net/Error.cs

```diff
@@ -1,27 +1,43 @@
 ﻿using System.Collections.Generic;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace EasyPost
 {
     public class Error : Resource
     {
         [JsonProperty("code")]
         public string code { get; set; }
+
         [JsonProperty("errors")]
-        public List<Error> errors { get; set; }
-        [JsonProperty("field")]
-        public string field { get; set; }
+        private List<JObject> errors
```

Comment:
I don't think this is what we want here. The `suggestions` error key typically only returns on address verification errors. It looks like this treats them as present on every error?

Reply:
Rolled back in fed1cecdc0406fa2d069b1b422f9d238927d4265, will address in a future PR